apps/web-login/js/moudle/constants.js

```diff
@@ -0,0 +1,9 @@
+const NAVER_CLIENT_ID = "Hw7RwBFOB7UjSufSEEjy"; // 상수는 따로 .js 파일에 넣어서 관리하자
+const KAKAO_CLIENT_ID = "13cee8f0d511dff11f766fe1d031b0a1";
+const cookieName = "userProfile";
+const NAVER_OAUTH_SECRETE = "MaLARH3T4B";
+sessionStorage.setItem("cookieName", `${cookieName}`);
+sessionStorage.setItem("NAVER_CLIENT_ID", `${NAVER_CLIENT_ID}`);
+sessionStorage.setItem("NAVER_OAUTH_SECRETE", `${NAVER_OAUTH_SECRETE}`);
+
+export { NAVER_CLIENT_ID, KAKAO_CLIENT_ID, cookieName, NAVER_OAUTH_SECRETE };
```

Comment:
이미 export 했는데 왜 상수를 세션 스토리지에 넣어서 사용하는걸까요?

Reply:
그리고 string 을 넣는데 왜 다시 ``로 덮어서 사용하는거죠?
```suggestion
sessionStorage.setItem("cookieName", cookieName);
sessionStorage.setItem("NAVER_CLIENT_ID", NAVER_CLIENT_ID);
sessionStorage.setItem("NAVER_OAUTH_SECRETE", NAVER_OAUTH_SECRETE);

export { NAVER_CLIENT_ID, KAKAO_CLIENT_ID, cookieName, NAVER_OAUTH_SECRETE };
```